### ISSUE TITLE
Adding node selector options

### ISSUE THREAD

**Issue by cskiwi** [kubero-dev/kubero#315]
hi,

I was wondering if we could have the `nodeSelector` option availible in the config of a deployment/stage/pipeline.
So we can run the apps on the right nodes :)

e.g.:
```yaml
nodeSelector:
  worker: 'true'
```

This would make sure it only runs on the nodes with the worker annotation on it

**Comment by mms-gianni** [comment]
Good point. There might be one already in the app CRD. But let me check that.

**Comment by mms-gianni** [comment]
There is a nodeSelector field for apps here: 

https://github.com/kubero-dev/kubero-operator/blob/main/helm-charts/kuberoapp/values.yaml#L162

But it's not acceisble by UI
